src/kernels/errors/invalidRemoteJupyterServerUriHandleError.ts

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Kernel related errors are moved into `src/kernels/errors` instead of having them in `src/platform`.